### DISCUSSION TITLE
how to perform vercel --force in GUI

### DISCUSSION THREAD

**Discussion by hussamkhatib** [vercel/vercel#8325] [Help] [answered]
Hey there, 
I am having a two projects deployed in a vercel project within the same repe.
I want to deploy both the project by skip the [build ca che](https://vercel.com/docs/cli#introduction/global-options/force).
The vercel CLI does the job for a single of it, I was not able to figure out how to do it for the other one.

In case you want to why I want to do it,
vercel is using old environment variables,I can confim by it as it worked for one of the project by clearing cache by CLI

**Comment by amyegan** [comment, marked as answer]
If you only need to skip the build cache one time to reset things, you can manually redeploy from the Dashboard. Go to your Project page, then the Deployments tab, then find the existing deployment that you want to redeploy and click the three dots to open a menu of actions. Click "redeploy" and a small form will pop up. Make sure that "Redeploy with existing Build Cache" is **not** checked and then click "Redeploy" to deploy the same build without the existing build cache.

![image](https://user-images.githubusercontent.com/8247105/182952733-8ce75e6a-5c39-43ce-8367-7fb98517527a.png)
